app/javascript/Components/Helpers/table_helpers.jsx

```diff
@@ -121,11 +121,13 @@ export function selectFilter({filter, onChange, column}) {
 
 export function markingStateColumn(marking_states, markingStateFilter, ...override_keys) {
   return {
-    Header: I18n.t("activerecord.attributes.result.marking_state"),
-    accessor: "marking_state",
-    Cell: row => {
+    header: I18n.t("activerecord.attributes.result.marking_state"),
+    accessorKey: "marking_state",
+    size: 100,
+    enableResizing: true,
+    cell: info => {
       let marking_state = "";
-      switch (row.original.marking_state) {
+      switch (info.row.original.marking_state) {
```

Comment:
Call the parameter `props`, not `info` (to match the other examples we have). 

Also, given the use of the `accessorKey`, it shouldn't be necessary to access `props.row.original.marking_state`, instead you should be able to use `props.getValue()`.